amethyst_renderer/src/cam.rs

```diff
@@ -53,25 +53,128 @@ impl From<Projection> for Matrix4<f32> {
 }
 
 /// Camera struct.
-///
-/// TODO: Add more convenience methods, refine API.
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct Camera {
+    /// Initial forward vector of the camera.
+    pub base_forward: Vector3<f32>,
```

Comment:
Can't we just make this `(0.0, 0.0, -1.0)`?

Reply:
The old implementation let the user set their own up- and right-vectors, so I just left it there. If we set the base-forward, we might as well define the up vector, too. Though we have to communicate that decision clearly in the book and docs.

I don't care either way, since I don't know of any specific use case, which would require the user to deviate (other than preference).

Reply:
I guess we shouldn't use these directional vectors any longer and use transform stuff instead.